internal/cli/plugin/plugin_github_asset.go

```diff
@@ -293,19 +291,15 @@ func (g *GithubAsset) verifyAssetSignature(asset []byte, sigAssetID, pubKeyAsset
 	return nil
 }
 
-func parseGithubReleaseValues(arg string) (*GithubAsset, error) {
-	regexPattern := `^((https?://(www\.)?)?github\.com/)?(?P<owner>[\w.\-]+)/(?P<name>[\w.\-]+)/?(@(?P<version>.+))?$`
-	regex, err := regexp.Compile(regexPattern)
-	if err != nil {
-		return nil, fmt.Errorf("error compiling regex: %w", err)
-	}
+var ghReleaseReg = regexp.MustCompile(`^((https?://(www\.)?)?github\.com/)?(?P<owner>[\w.\-]+)/(?P<name>[\w.\-]+)/?(@(?P<version>.+))?$`)
```

Comment:
drive-by, regex should be compiled once if possible because is an expensive operation